src/site/content/en/blog/image-policies/index.md

```diff
@@ -0,0 +1,265 @@
+---
+title: Image policies for fast load times and more
+subhead: Use optimized image policies to ensure your site is using the best performing images.
+authors:
+  - lunalu
+date: 2019-05-23
+codelabs:
+  - codelab-serve-images-webp
+hero: pankaj-patel-721645-unsplash.jpg
+alt: A description of the hero image for screen reader users.
+description: |
+  Images take up a significant amount of visual space and make up the
+  majority of the downloaded bytes on a website. Use the new feature
+  policies to identify oversized images.
+tags:
+  - post
+  - images
+  - origin-trials
+---
+Images often take up a significant amount of visual space and make up the
+majority of the downloaded bytes on a website. Optimizing images can improve
+loading performance and reduce network traffic.
+
+Surprisingly, more than half of the sites on the web are serving poorly
+compressed or unnecessarily large images. This leaves a lot of room for
+performance improvements simply by optimizing the images.
+
+You may ask, how do I know if my images are optimized and how should I optimize
+them? We are experimenting with a new set of feature policies for image
+optimization: `oversized-images`, `unoptimized-lossy-images`,
+`unoptimized-lossless-images`, and`unoptimized-lossless-images-strict`.
+All are now available for [origin
+trials](https://github.com/GoogleChrome/OriginTrials/blob/gh-pages/developer-guide.md).
+
+## Optimized image policies
+
+Feature policy is introducing a new set restrictions on images that can be
+applied with development-time enforcement. Images violating any of the
+restrictions will be rendered as placeholder images, which are easy to identify
+and fix. These policies can be specified in report-only mode where images will
+render normally without enforcement while violations are being observed via
+reports. (See [Report-only mode in the wild](#report-only-mode-in-the-wild),
+below for details.)
+
+### oversized-images
+The `oversized-images` feature policy restricts the intrinsic dimensions
+of an image in relation to its container size.
+
+When a document uses the `oversized-images` policy, any `<img>` element
+whose intrinsic resolution is more than X times larger than the container size
+in either dimension will be replaced with a placeholder image.
+
+
+#### Why?
+
+Serving images larger than what the viewing device can render&mdash;for example,
+serving desktop images to mobile contexts, or serving high-pixel-density images
+to a low-pixel-density device&mdash;is wasting network traffic and device memory.
```

Comment:
Can we add here:
Read [Serve images with correct dimensions](https://web.dev/serve-images-with-correct-dimensions/) and [Serve responsive images](https://web.dev/serve-responsive-images/) on how to optimize your images.

Reply:
Done.